app/controllers/thread_controller.php

```diff
@@ -0,0 +1,62 @@
+<?php
+class ThreadController extends AppController 
+{
+    function __construct($name)
+    {
+        parent::__construct($name);
+
+       if (!is_logged_in())
+       {
+            redirectTo(url('user/login'));
+       }
+    }
+
+    public function index()
```

Comment:
Same as this [comment](https://github.com/edwinflores/board_example/pull/4/files#r22446580).